src/dredd-transactions.coffee

```diff
@@ -1,36 +1,22 @@
 
 parse = require('./parse')
-compileFromApiBlueprintAst = require('./from-api-blueprint-ast/compile')
-compileFromApiElements = require('./from-api-elements/compile')
+compileFromApiElements = require('./compile')
```

Comment:
I'm a bit grappling with the name since the main interface of the whole library is also `compile`, so variable shadowing would happen here. I deferred the solution for now, but I'd like to make the names distinct & consistent in the future.